.github/workflows/vercel-ci.yml

```diff
@@ -32,4 +32,10 @@ jobs:
       # NOTE Since build is automated by vercel, it's probably
       # redundant to test it here, too
       - name: Svelte Check
-        run: npm run check -- --fail-on-warnings --fail-on-hints
+        run: npm run check:svelte -- --fail-on-warnings --fail-on-hints
+
+      - name: ESLint
+        run: npm run check:lint
+
+      - name: Prettier
+        run: npm run check:prettier
```

Comment:
Will the CI fail on warnings?

Reply:
`check:prettier` has exit code 1 on warnings. Warnings have exit code 0 with ESLint, but we can use `--max-warnings=0` if we want to be strict about warnings.

Reply:
Got it, ESLint does return an exit code of 1 on errors, but not on warnings:

<img width="822" alt="Screenshot 2022-08-03 at 01 18 12" src="https://user-images.githubusercontent.com/13710835/182492599-12a20f4e-7588-4dbb-8532-21a1067b1d88.png">